app/views/full-page-examples/localisation/index.njk

```diff
@@ -0,0 +1,233 @@
+
+        <div role="note" class="govuk-inset-text">
+          <p class="govuk-body">Ni allwch wneud cais mewn Swyddfa’r Post os yw’ch enw wedi newid. Bydd angen ichi wneud cais drwy’r post.</p>
+        </div>
+
+        <h2 class="govuk-heading-m" id="gwneud-cais-drwyr-post">Gwneud cais drwy’r post</h2>
+
+        <div role="note" class="govuk-inset-text">
+          <p class="govuk-body">Mae’n cymryd yn hirach na’r arfer i brosesu ceisiadau a anfonir trwy’r post oherwydd coronafeirws (COVID-19).</p>
+        </div>
+
+        <p class="govuk-body">Gallwch gael ‘pecyn D1W’ o ffurflenni o <a class="govuk-link" rel="external" href="http://www.postoffice.co.uk/branch-finder">Swyddfa’r Post</a> sy’n delio ag adnewyddu trwyddedau cerdyn-llun neu dreth cerbyd DVLA.</p>
+
+        <p class="govuk-body">Mae angen ichi gynnwys y pethau canlynol gyda’ch ffurflenni wedi’u cwblhau:</p>
+
+        <ul class="govuk-list govuk-list--bullet">
+          <li>
+        <a class="govuk-link" href="/photos-for-passports">ffotograff math pasbort</a> diweddar (peidiwch â llofnodi cefn y llun)</li>
+          <li>eich trwydded cerdyn-llun gyfredol os ydyw gennych</li>
+          <li>siec neu archeb bost am £17, yn daladwy i DVLA (nid oes ffi yn daladwy os oes gennych drwydded yrru cyfnod byr am resymau meddygol neu os ydych yn 70 oed neu’n hŷn)</li>
+        </ul>
+
+        <p class="govuk-body">Mae hefyd angen ichi gynnwys <a class="govuk-link" href="/driving-licence-application-identity-documents">dogfennau adnabod</a> os ydych wedi newid eich enw.</p>
+
+        <p class="govuk-body">Anfonwch y cais i:</p>
+
+        <p class="govuk-body">DVLA
+        <br>Abertawe
+        <br>SA99 1DH
+        <br>
+        </p>
+
+        <h2 class="govuk-heading-m" id="ar-l-ichi-wneud-cais-mewn-swyddfar-post-neu-drwyr-post">Ar ôl ichi wneud cais mewn Swyddfa’r Post neu drwy’r post</h2>
+
+        <p class="govuk-body">Gallwch barhau i yrru tra rydych yn aros am eich trwydded newydd gyrraedd.</p>
+
+      {% endset %}
+
+      {{ govukAccordion({
+        id: "accordion-default",
+        items: [
+          {
+            heading: { text: "Rhagor o wybodaeth" },
+            content: {
+              html: moreInformationHTML
+            }
+          },
+          {
+            heading: { text: "Ffyrdd eraill o wneud cais" },
+            content: {
+              html: otherWaysToApplyHTML
+            }
+          }
+        ]
+      }) }}
+    </div>
+
+    <div class="govuk-grid-column-one-third">
+
+      <!-- The Related items component is not part of GOV.UK Frontend but will be styled if used in the Prototype Kit -->
+
+      <aside class="app-related-items" role="complementary">
+        <h2 class="govuk-heading-m" id="subsection-title">
+          Cynnwys cysylltiedig
+        </h2>
+        <nav role="navigation" aria-labelledby="subsection-title">
+          <ul class="govuk-list govuk-!-font-size-16">
+            <li class="gem-c-related-navigation__link">
```

Comment:
Think these gem-c classes have snuck in.

Reply:
I copied this page from one of the other full-page examples, though I'm happy to remove given it's pretty much decorative here.